python/src/MonteCarloLHS_doc.i.in

```diff
@@ -3,24 +3,22 @@
 
 Performs the optimization of an LHS using Monte Carlo simulations.
 
-Available constructors:
-    MonteCarloLHS(*lhsDesign, N*)
-
-    MonteCarloLHS(*lhsDesign, N, spaceFilling*)
-
 Parameters
 ----------
 lhsDesign : :class:`~openturns.LHSExperiment`
      Factory that generate designs
 N : int
     Number of simulations
-spaceFilling : :class:`~openturns.SpaceFilling`
+spaceFilling : :class:`~openturns.SpaceFilling`, optional
     Criterion to be optimized
```

Comment:
```suggestion
    Criterion to be optimized.
```

Reply:
It's not really a sentence, so I think a dot is optional here

Reply:
@jschueller The issue is that it looks weird on the HTML output since there is no carriage return:

![image](https://user-images.githubusercontent.com/31989332/155088795-dc6e787b-9fa3-4816-82cf-4748bdc83acf.png)

Reply:
ok